### PR TITLE
[Website/docs]: Adds "Using Fe" section

### PR DESCRIPTION
**Pull request by jmcook1186**
Adds "Using Fe" section to the docs. This is where we can collect end-user info such as the installation guide, info about using projects, etc. This is also where we can have a sub-directory for example app walkthrough guides.

Moved the installation and projects pages out of quickstart and into the new Using Fe section.

### CODE REVIEW

File: docs/src/quickstart/index.md [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+In this section you will learn how to write and deploy your first contract.
 
-* [Installation](installation.md)
 * [Writing your first contract](first_contract.md)
 * [Deploying a contract to a testnet](deploy_contract.md)
-* [Using projects](projects.md)
+
+## Download and install Fe
+
+Before you dive in, you need to download and install Fe.
```
Comment (cburgdorf) [inline docs/src/quickstart/index.md:12]: Not necessary to update this PR but I just wanted to make you aware that we recently (basically yesterday) automated our homebrew installation mechanics. Basically if you have homebrew installed (Mac or Windows) you can run `brew install fe-lang/tap/fe` now to install fe and expose it as `fe` command without any further hassle.

We don't have to maintain these manually so they should automatically keep in sync as we cut new releases.

**Review by cburgdorf — APPROVED** [review]

**Comment by cburgdorf** [comment]
Github says I can't merge it because it needs to have conflicts resolved.

**Comment by jmcook1186** [comment]
conflicts should be resolved now..?

File: docs/src/quickstart/index.md (hunk 1 above)
Comment (jmcook1186) [reply docs/src/quickstart/index.md:12]: oh that's awesome. I wasn;t aware - ok I will add this information to this page and the installation guide in the next pr.

**Comment by cburgdorf** [comment]
> conflicts should be resolved now..?

image

It isn't. You should fetch latest master and then rebase on top of latest master, resolving conflicts as they appear. Let me know if you run into trouble with that, I can also handle the conflict resolution on my side prior to merging.

**Comment by cburgdorf** [comment]
Hey @jmcook1186 the PR still had conflicts and hence Github would not let me merge it. Here is what I did to resolve it.

1. I checked out your PR locally and rebased it on top of latest master. This process will take each of your commits and perform it on top of latest master stopping at commits that produce conflicts

image

`git status` will tell me which files have conflicts that need to be resolved.

image

My edit tells me what the conflict is and lets me resolve it.

image

 Once all conflicts are resolved you stage the changes via `git add .` (not in the screenshot) and run `git rebase --continue`

image

This process might repeat if there continue to be conflicts in the commits that follow to be applied on top.

I did all that, pushed it into your PR and now it can be merged. Hope that helps!

**Pull request merged**